src/plugins/vis_type_timeseries/server/lib/vis_data/response_processors/table/math.js

```diff
@@ -8,9 +8,9 @@
 
 import { mathAgg } from '../series/math';
 
-export function math(bucket, panel, series) {
+export function math(bucket, panel, series, meta, extractFields) {
```

Comment:
a separate PR (#100765) was created for that change which should be backported into `7.13.2`